<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-08-19T13:18:31 IST" name="Groupig_Suite" finished-at="2025-08-19T13:19:57 IST" duration-ms="86623">
    <groups>
    </groups>
    <test started-at="2025-08-19T13:18:31 IST" name="Group_Test" finished-at="2025-08-19T13:19:57 IST" duration-ms="86623">
      <class name="TestCase.TC002_LoginTest">
        <test-method is-config="true" signature="setUp(java.lang.String)[pri:0, instance:TestCase.TC002_LoginTest@13bc8645]" started-at="2025-08-19T13:19:33 IST" name="setUp" finished-at="2025-08-19T13:19:51 IST" duration-ms="18346" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="verify_login()[pri:0, instance:TestCase.TC002_LoginTest@13bc8645]" started-at="2025-08-19T13:19:51 IST" name="verify_login" finished-at="2025-08-19T13:19:57 IST" duration-ms="6539" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verify_login -->
      </class> <!-- TestCase.TC002_LoginTest -->
      <class name="TestCase.TC001_AccRegTest">
        <test-method is-config="true" signature="setUp(java.lang.String)[pri:0, instance:TestCase.TC001_AccRegTest@19b89d4]" started-at="2025-08-19T13:18:31 IST" name="setUp" finished-at="2025-08-19T13:19:20 IST" duration-ms="49469" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="verify_Acc_Registration()[pri:0, instance:TestCase.TC001_AccRegTest@19b89d4]" started-at="2025-08-19T13:19:20 IST" name="verify_Acc_Registration" finished-at="2025-08-19T13:19:33 IST" duration-ms="12204" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at TestCase.TC001_AccRegTest.verify_Acc_Registration(TC001_AccRegTest.java:67)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verify_Acc_Registration -->
      </class> <!-- TestCase.TC001_AccRegTest -->
    </test> <!-- Group_Test -->
  </suite> <!-- Groupig_Suite -->
</testng-results>
